avocado/utils/multipath.py

```diff
@@ -106,6 +106,19 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "egrep -i mpathd /etc/multipath/bindings"
+    wwid = process.run(cmd, ignore_status=True,
+                        sudo=True, shell=True).stdout_text
+    return wwid.split()[1]
```

Comment:
this should be inside a try.. except

Reply:
Check is done before the value is returned.